interfaces/cython/cantera/reaction.pyx

```diff
@@ -279,9 +279,9 @@ cdef class TwoTempPlasmaRate(ArrheniusTypeRate):
 
         k_f = A T_e^b \exp(-\tfrac{E_{a,g}}{RT}) \exp(\tfrac{E_{a,e}(T_e - T)}{R T T_e})
 
-    where :math:`A` is the `pre_exponential_factor`, :math:`b` is the
-    `temperature_exponent`, :math:`E_{a,g}` is the `activation_energy`, and
-    :math:`E_{a,e}` is the `activation_electron_energy`.
+    where ``A`` is the `pre_exponential_factor`, ``b`` is the
+    `temperature_exponent`, ``E_{a,g}`` is the `activation_energy`, and
+    ``E_{a,e}`` is the `activation_electron_energy`.
```

Comment:
I guess this ends up being a bit confusing, but if these are going to be formatted like `code`, then I think the names should correspond to the argument names in `__cinit__` below (and correspondingly, as this is written in `kinetics.rst`, e.g. `Ea_electron`, since this documentation will appear below that constructor signature. But that does kind of conflict with the usual description of the mathematical notation. 🤷

I also realized from looking at https://cantera.org/documentation/dev/sphinx/html/cython/kinetics.html#twotempplasmarate that Sphinx is apparently not smart enough to link to properties defined in a base class, which is a bit unfortunate, so the formatting of `pre_exponential_factor`, `temperature_exponent`, and `activation_energy` doesn't look great, while `activation_electron_energy` is a link as I would have expected for all four.

Reply:
Some of the Sphinx issues were introduced by introducing an inheritance structure (which I believe was after the class was initially introduced). There may be ways to tell Sphinx what to do by adding the base class? Fwiw, the formatting can be also checked from the artifacts for each successful GH run.

Reply:
Fwiw, `` `pre_exponential_factor <ArrheniusTypeRate.pre_exponential_factor>` `` works ...